shared/img/esbuild.svg

```diff
@@ -0,0 +1,30 @@
+        <linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="linearGradient-4">
+            <stop stop-color="#FFC700" offset="0%"></stop>
+            <stop stop-color="#FF9600" offset="100%"></stop>
+        </linearGradient>
+    </defs>
+    <g id="ESbuild" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group">
+            <g id="Polygon" transform="translate(82.000000, 82.011168) rotate(-210.000000) translate(-82.000000, -82.011168) ">
+                <use stroke="#FFFFFF" stroke-width="18" fill="url(#linearGradient-1)" fill-rule="evenodd" xlink:href="#path-3"></use>
+                <use stroke="url(#linearGradient-2)" stroke-width="10" xlink:href="#path-3"></use>
+            </g>
+            <path d="M91.090898,4.19025709 C92.0788464,3.92211961 93.1690857,4.03087173 94.127447,4.58006893 C94.8090191,4.97064975 95.3447093,5.54264544 95.6912809,6.21248946 C96.0378525,6.88233348 96.1953055,7.65002584 96.1204,8.43203519 L96.1204,8.43203519 L90.5083787,67.0103206 L122.274381,67.0111676 C123.102808,67.0111676 123.852808,67.346954 124.395701,67.8898472 C124.938594,68.4327404 125.274381,69.1827404 125.274381,70.0111676 C125.274381,70.5344973 125.137482,71.0487235 124.87728,71.5027818 L124.87728,71.5027818 L75.331905,157.960552 C74.7827078,158.918913 73.8970504,159.563941 72.909102,159.832078 C71.9211536,160.100216 70.8309143,159.991463 69.872553,159.442266 C69.1909809,159.051685 68.6552907,158.47969 68.3087191,157.809846 C67.9621475,157.140002 67.8046945,156.372309 67.879594,155.590363 L67.879594,155.590363 L73.4899186,97.0103206 L41.7256194,97.0111676 C40.8971923,97.0111676 40.1471923,96.6753811 39.6042991,96.1324879 C39.0614058,95.5895947 38.7256194,94.8395947 38.7256194,94.0111676 C38.7256194,93.4878378 38.8625176,92.9736116 39.1227197,92.5195533 L39.1227197,92.5195533 L88.668095,6.06178287 C89.2172922,5.10342166 90.1029496,4.45839458 91.090898,4.19025709 Z" id="Combined-Shape" stroke="#FFFFFF" stroke-width="4" fill="url(#linearGradient-4)"></path>
+            <polygon id="Star" stroke="#FFFFFF" stroke-width="18" transform="translate(82.000000, 82.011168) rotate(-90.000000) translate(-82.000000, -82.011168) " points="82 82.0111676 40.4307806 106.011168 82 82.0111676 82 34.0111676 82 82.0111676 123.569219 106.011168"></polygon>
+            <polygon id="Star" stroke="url(#linearGradient-2)" stroke-width="10" transform="translate(82.000000, 82.011168) rotate(-210.000000) translate(-82.000000, -82.011168) " points="82 82.0111676 34.3686028 109.511168 82 82.0111676 82 27.0111676 82 82.0111676 129.631397 109.511168"></polygon>
+        </g>
+    </g>
+</svg>
```

Comment:
Reminder: Need to optimise this before landing

Reply:
minified with svgo